apps/api/app/agents/middleware/subagent.py

```
@@ -0,0 +1,394 @@
+                )
+                return Command(
+                    update={
+                        "messages": [
+                            ToolMessage(
+                                content=result,
+                                tool_call_id=tool_call_id,
+                            )
+                        ]
+                    }
+                )
+            except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)